py/miniconf-mqtt/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[license]
+text = "MIT"
+
+[project]
+name = "miniconf-mqtt"
+# Note: keep this in sync with Cargo.toml
+version = "0.11.0"
+#url = "https://github.com/quartiq/miniconf"
```

Comment:
Can't we have a url?

Reply:
I simply forgot to fix this after initially commenting it out. Added the proper URLs section for the package in 51d73266a37342b6646bcd9aafaab9c4b9edb983